docs/bugpattern/ChainedAssertionLosesContext.md

```diff
@@ -6,7 +6,7 @@ Before:
 ```
 class MyProtoSubject {
   public void hasFoo(Foo expected) {
-    assertThat(actual().foo()).isEqualTo(expected);
+    assertThat(actual.foo()).isEqualTo(expected);
```

Comment:
AFAICT, `MyProtoSubject` lacks an `actual` variable or field, so I wonder if it should be updated? :thinking:

Reply:
The class is missing a constructor and lots of other things, too. I don't want there to be too much noise around the actual bug, but I should probably insert a "..." in there to make that clear. Hmm, and `extends Subject`, too. Will do. thanks